backend/src/app.ts

```diff
@@ -69,6 +73,37 @@ app.use('/api/v1/stripe', stripeRouter);
 app.use('/api/v1/integration', integrationRouter);
 app.use('/api/v1/integration-auth', integrationAuthRouter);
 
+
+//* Handle unrouted requests and respond with proper error message as well as status code
+app.use((req, res, next)=>{
```

Comment:
Would be great if you could modularize this last middleware into a separate file as it feels a little chunky here.

Reply:
No problem.

Reply:
Split error handler middleware to a seperate file as your request.

Reply:
Oh no I messed up  the naming :|